appveyor.yml

```diff
@@ -32,7 +32,7 @@ environment:
 build_script:
   - if [%COMPILER%]==[MinGW-w64] mkdir bin && cd bin\ && appveyor DownloadFile http://www.tortall.net/projects/yasm/releases/yasm-1.3.0-win64.exe -fileName yasm.exe
   - if [%COMPILER%]==[MinGW-w64] C:\msys64\usr\bin\sh.exe --login /c/projects/mpir/.appveyor_msys_build.sh
-  - if [%COMPILER%]==[MSVC15] cd build.vc14
+  - if [%COMPILER%]==[MSVC15] cd msvc\vs17
```

Comment:
change this to `msvc\vs15` as the job is named `MSVC15`.

Reply:
@isuruf - this is one of the quirky things about VisualStudio - the compiler versions are independent of the visual studio version, and as fate would have it, the version numbers are confusingly close to the year numbers; the MSVC15 compiler corresponds to Visual Studio 2017.  Personally when I am searching for which project to open I find the visual studio year easier to identify, so I think the vs13, vs15, and vs17 names @BrianGladman chose are good ones.

Reply:
Okay. Can you add `image: Visual Studio 2017` to the top of `appveyor.yml` so that an image with Visual Studio 2017 is used?